gamemodes/zombieplague/gamemode/sv_eventmanager.lua

```diff
@@ -425,12 +425,14 @@ timer.Create("SecondTickManager", 1, 0, function()
 	hook.Call("SecondTickManager")
 end)
 net.Receive("RequestSpectator", function(len, ply)
-	if ply:Team() != TEAM_SPECTATOR then
-		RoundManager:RemovePlayerToPlay(ply)
-		ply:KillSilent()
-		ply:Spectate(OBS_MODE_ROAMING)
-	else
-		RoundManager:AddPlayerToPlay(ply)
+	if RoundManager:GetRoundState() != ROUND_PLAYING then
```

Comment:
It is a good ideia avoiding spectator during round? I mean, what if for some reason I want to? Or I need to?